### DISCUSSION TITLE
Z2M update no response for 1.42

### DISCUSSION THREAD

**Discussion by wildchai** [community-scripts/ProxmoxVE#799] [Q&A] [answered]
Hi everyone,

I've been consistently updating my Z2M via the script over the year. But comes to 1.42, the code does not work. Running it doesn't do anything. Tried restarting the LXC and Proxmox still the same. 

Currently on 1.40.2

I ran this in LXC console : bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/zigbee2mqtt.sh)"

Please advise. Thanks.

**Comment by MickLesk** [comment, marked as answer]
Read discussions
https://github.com/community-scripts/ProxmoxVE/discussions/660

**Reply by wildchai** [reply]
Thanks. Will review and post results.

**Reply by wildchai** [reply]
Managed to have time to update to 2.1
Hit a few issues....config yaml was missing, pnpm unable to install, files not committed....after all those.

Guide was useful and working now. thanks.
